source/manual/seeing-things-in-the-aws-console.html.md

```diff
@@ -0,0 +1,63 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Seeing things in the AWS Console
+section: AWS
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-06-01
+review_in: 3 months
+---
+
+To follow this guide, you'll need a `gds-users` AWS account. Follow
+[the User Management in AWS guide](source/manual/user-management-in-aws.html.md)
+to get one.
+
+## Seeing things in the AWS Console
+
+The AWS Console can seem impenetrable, with lots of options, some of
+them restricted by your role access level.
+
+### Logging in to the central `gds-users` account
+
+To log into the central `gds-users` account, [go to the AWS Console](https://console.aws.amazon.com/).
+
+Enter `gds-users` as the "IAM user account alias".
+
+![aws-console-sign-in](images/aws-console-sign-in.png)
+
+Next, enter your email address and password. After clicking "Sign in",
+you'll be prompted for your MFA token.
+
+Once you're logged in, you'll see a page that looks like a normal AWS
+console.
+
+![gds-users-aws-console](images/gds-users-aws-console.png)
+
+If you try to click on anything here, for example EC2, you'll get "You
+are not authorized". This is because you're still in the `gds-users`
+account which is locked down. To get to the GOV.UK accounts, where you
+have permissions, you'll need to "Switch Role".
+
+### Switching role to one of the GOV.UK accounts
+
+1. Click "Switch Role" in the top right corner.
+
+  ![gds-users-switch-role](images/gds-users-switch-role.png)
+
+2. Fill in the account alias (this can be found
+  [in the govuk-aws-data repo](https://github.com/alphagov/govuk-aws-data/blob/master/docs/govuk-aws-accounts.md)),
+  the role you're assuming (`govuk-users`, `govuk-poweruser`, or
+  `govuk-administrators`), and a name for that role (`govuk-<environment>`).
+
+  ![gds-users-switch-role-form](images/gds-users-switch-role-form.png)
+
+3. Submit the form, and you'll now be in one of the GOV.UK accounts
```

Comment:
There's an extra step here - possible the most confusing part - to switch region. Is there a way to know which region we're in? Perhaps this could go on that page of aliases?

Reply:
Ah yes! That was in my head and then I got distracted taking screenshots!

Reply:
Also, it would be useful to clarify how you know which role you are in. These are some of the hardest files to find ;-)